pom.xml

```
@@ -196,14 +196,14 @@
             <dependency>
                 <groupId>org.springframework</groupId>
                 <artifactId>spring-framework-bom</artifactId>
-                <version>5.1.8.RELEASE</version>
+                <version>5.3.9</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
             <dependency>
                 <groupId>org.junit</groupId>
                 <artifactId>junit-bom</artifactId>
-                <version>5.5.0</version>
+                <version>5.7.2</version>
```

Sett denne på `5.8.0-RC1` like gjerne. Det funker fjell!
```suggestion
                <version>5.8.0-RC1</version>
```

---

heh, jeg bare gønner på, jeg 🤠